core/server/api_container/server/startosis_engine/startosis_interpreter_plan_yaml_test.go

```diff
@@ -813,6 +833,9 @@ tasks:
   command:
   - echo {{ kurtosis.4.code }} {{ kurtosis.4.output }}
   image: badouralix/curl-jq
+images:
+- badouralix/curl-jq
+- postgres:latest
 `
 	require.Equal(suite.T(), expectedYaml, planYaml)
 }
```

Comment:
Should there be a test case here where the `dependenciesOnly` flag is `true`? (assuming that, per the comment above, we maintain the `dependenciesOnly` flag on the PlanYamlGenerator and it's not the responsibility of the CLI or user to pull out the info they want)

Reply:
removed the boolean so shouldn't need to test that case